src/content/docs/en/guides/fonts.mdx

```diff
@@ -83,42 +75,43 @@ The [Fontsource](https://fontsource.org/) project simplifies using Google Fonts
     </PackageManagerTabs>
 
     :::tip
-    You’ll find the correct package name in the “Quick Installation” section of each font page on Fontsource’s website. It will start with `@fontsource/` followed by the name of the font.
+    You'll find the correct package name in the “Quick Installation” section of each font page on Fontsource's website. It will start with `@fontsource/` followed by the name of the font.
     :::
 
-3. Import the font package in the layout or component where you want to use the font. Usually, you will want to do this in a common layout component to make sure the font is available across your site.
+1. Import the font package in the layout or component where you want to use the font. Usually, you will want to do this in a common layout component to make sure the font is available across your site.
 
     The import will automatically add the necessary `@font-face` rules needed to set up the font.
 
-    ```astro
+    ```astro title="src/layouts/BaseLayout.astro"
     ---
-    // src/layouts/BaseLayout.astro
     import '@fontsource/twinkle-star';
     ---
     ```
 
-4. Use the `font-family` as shown on that font’s Fontsource page. This will work anywhere you can write CSS in your Astro project.
+1. Use the `font-family` as shown on that font's Fontsource page. This will work anywhere you can write CSS in your Astro project.
```

Comment:
```suggestion
1. Use the font's name as shown in the `body` example on its Fontsource page as the `font-family` value. This will work anywhere you can write CSS in your Astro project.
```
I checked the website for clarity, since the package name (kebab-case) is also shown "on the font's Fontsource page". I think something like more this hits home that use import the package name, but you *use* the font familiar name in the CSS directly?

Reply:
Sorry, had a little typo:

The original wording said "as shown on the Fontsource page" but on that page itself, it shows both the package name (`twinkle-star`) AND the regular, natural language name `Twinkle Star`. So my objection was re: just saying to use it "as shown on the page" because that's not specific enough: it's shown in two different ways there.

The nature of my change was to call attention to the difference: that when you import, you use the *package* name (kebab case), but when you actually write in the body of the CSS, you use the *font's* "given" name (for lack of a better word). 

Does my suggested edit make sense?

Reply:
Thanks for the clarification.
Yes, your suggestion does make sense, and after checking the quick installation page, I had another thought. In the suggested edit, we kind of committed to the way how installation looks like. What if, at some point in the future, they will update the instructions? Or we are ok with updating the page afterward?